src/games/seabattle/game.ts

```diff
@@ -53,6 +54,9 @@ export const SeabattleGame = Game({
   moves: {
     setShips(G: ISeabattleState, ctx: ICtx, ships: IShip[]) {
       const player = parseInt(ctx.playerID, 10);
+      for (let i = 0; i < ships.length; i++) {
+        ships[i] = { ...ships[i], id: i + G.ships.length };
```

Comment:
Are you overriding the random Id you generated in the random ship functions? Are you sure this works because this is relying on G which will be different for each player?

Reply:
fixed by using shortids